src/main/java/com/nha/abdm/wrapper/hrp/repository/PatientRepo.java

```diff
@@ -4,9 +4,11 @@
 import org.springframework.data.mongodb.repository.MongoRepository;
 import org.springframework.stereotype.Repository;
 
+import java.util.List;
+
 @Repository
 public interface PatientRepo extends MongoRepository<Patients, String> {
     Patients findByAbhaAddress(String abhaAddress);
     Patients findByPatientReference(String patientReference);
-    Patients findByPatientMobile(String patientMobile);
+    List<Patients> findByPatientMobile(String patientMobile);
```

Comment:
Rename to `findByMobileNumber(String mobileNumber)`

Reply:
renamed the parameter name to findByMobileNumber()